cli/commands/build.py

```diff
@@ -378,7 +378,22 @@ def build_solace_agent_mesh(config, build_config_dir, abort, parsers):
         if f.endswith(".yaml")
         and not any(f.startswith(prefix) for prefix in skip_prefixes)
     ]
+    
+    # Check if embedding service is enabled
+    embedding_enabled = True  # Default to True for backward compatibility
```

Comment:
For backward compatibility, if the embedding service is not adding, we add it automatically. This will make sure that existing installations (Solace Chat) still work like before.